### ISSUE TITLE
Polycrystalline solar cell defect edge (PSCDE) dataset 光伏电池缺陷分割数据集！PS逐像素标注！

### ISSUE THREAD

**Issue by wch313** [Charmve/Surface-Defect-Detection#29]
We construct a polycrystalline solar cell defect edge (PSCDE) dataset, which is the first high-quality solar cell segmentation dataset. We adopt the electroluminescence imaging technique collecting 700 challenging defect images with 512×512 resolution, such as multi-scale defects, occlusion defects, dense tiny defects, low contrast defects, and combination defects. Furthermore, it is a comprehensive dataset containing the main defect categories, including linear cracks, star cracks, scratches, finger interruptions, black cores, fragments, and corners. Moreover, we meticulously annotate all defective edges with the help of multiple experts. The ground truth of PSCDE includes two categories: edge and non-edge. We randomly divide PSCDE into 280 images for the training set, 140 images for the validation set, and 280 images for the testing set and guarantee that each set contains all kinds of defects.
[https://github.com/wch313/PSCDE-Dataset](url)
![PSCDE](https://github.com/Charmve/Surface-Defect-Detection/assets/74942694/ed8b00ef-bf4f-40da-98ba-c19db70892e0)

**Comment by Charmve** [comment]
Great! Would you like to make a PR to this repo? @wch313
